docs/designs/2026-05-24-v2.8.0-closeout.md

```
@@ -0,0 +1,140 @@
+
+- **Date:** 2026-05-24
+- **Status:** Design — pending plan
+- **Milestone:** Strategos 2.8.0 — Cross-product schema substrate
+- **Feature ID:** `v2-8-0-closeout`
+- **Closes:** #63 (S1), #64 (S2), #65 (S4), #66 (S3), #104, #105, #106, #107, #54 (verify) — defers #100
+- **Lenses applied:** `/strategos-design-invariants` (INV-1, INV-2, INV-3, INV-5, INV-7, INV-8) · `/axiom:design` (DIM-1, DIM-2, DIM-3, DIM-4, DIM-5, DIM-6)
+
+## 1. Problem
+
+Slice A (`Strategos.Contracts` 0.2.0 — events / workflow IR / invariant catalog) shipped via
+#99/#101/#102/#103. The 2.8.0 milestone has 10 open issues that, taken together, are the
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)